File: VB/MainViewModel.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Core
Imports DevExpress.Mvvm
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Public Class MainViewModel
	Inherits ViewModelBase

	Public Sub New()
		Me.ColorSchemeName = Me.ColorSchemeNames(0)
	End Sub

	Protected _ColorSchemeNames As List(Of String)
	Public ReadOnly Property ColorSchemeNames() As List(Of String)
		Get
			If Me._ColorSchemeNames Is Nothing Then
				Me._ColorSchemeNames = New List(Of String) (New String() {ThemeManager.GenericSchemeName, "Custom"})
			End If
			Return Me._ColorSchemeNames
		End Get
	End Property

    Protected _Themes As List(Of ElementTheme)

    Public ReadOnly Property Themes() As List(Of ElementTheme)
	    Get
		    If Me._Themes Is Nothing Then
			    Me._Themes = New List(Of ElementTheme)() From { _
				    ElementTheme.Light, _
				    ElementTheme.Dark _
			    }
		    End If
		    Return Me._Themes
	    End Get
    End Property

	Public Property ColorSchemeName() As String
		Get
			Return ThemeManager.CurrentColorSchemeName
		End Get
		Set(ByVal value As String)
			ThemeManager.CurrentColorSchemeName = value
            RaisePropertyChanged("ColorSchemeName")
		End Set
	End Property


End Class

File: CS/MainViewModel.cs
using DevExpress.Core;
using DevExpress.Mvvm;
using System.Collections.Generic;
using Windows.UI.Xaml;

namespace ColosSchemesDemo1
{
    public class MainViewModel : ViewModelBase {

        public MainViewModel() {
            this.ColorSchemeName = this.ColorSchemeNames[0];
        }

        protected List<string> _ColorSchemeNames;
        public List<string> ColorSchemeNames {
            get {
                if (this._ColorSchemeNames == null) {
                    this._ColorSchemeNames = new List<string>() { ThemeManager.GenericSchemeName, "Custom" };
                }
                return this._ColorSchemeNames;
            }
        }

        protected List<ElementTheme> _Themes;

        public List<ElementTheme> Themes {
            get {
                if (this._Themes == null) {
                    this._Themes = new List<ElementTheme>() { ElementTheme.Light, ElementTheme.Dark };
                }
                return this._Themes;
            }
        }

        public string ColorSchemeName {
            get {
                return ThemeManager.CurrentColorSchemeName;
            }
            set {
                ThemeManager.CurrentColorSchemeName = value;
                this.RaisePropertyChanged("ColorSchemeName");
            }
        }


    }
}
